README.md

```diff
@@ -62,13 +63,24 @@ function (value) {
 Example usages
 
 Ask for a name:
+
 ```js
 promptly.prompt('Name: ', function (err, value) {
     // err is always null in this case, because no validators are set
     console.log(value);
 });
 ```
 
+Using Promise:
```

Comment:
Could you please remove all the "Using promise:" examples and just mention it once? perhaps at the top saying that all functions work with promises?